kodilla-testing/src/main/java/com/kodilla/testing/collection/OddNumbersExterminator.java

```diff
@@ -0,0 +1,36 @@
+package com.kodilla.testing.collection;
+import java.util.*;
+import com.google.common.collect.*;
+public class OddNumbersExterminator {
+
+    ArrayList<Integer>  exterminate(ArrayList<Integer> numbers){
+     ArrayList<Integer> arr=numbers;
+     ArrayList<Integer> arr2=Lists.newArrayList();
+    if (arr.size()==0){
+        arr=null;
+    }else {
+
+        for (int i = 0; i < arr.size(); i++) {
+
+            if ((arr.get(i) % 2) == 0) {
+                if (!arr2.contains(arr.get(i))) {
+
+                    arr2.add(arr.get(i));
+                }
+            }
+
+        }
+
+
+        for (int i = 0; i < arr2.size(); i++) {
+            while (arr.remove(arr2.get(i))) {
```

Comment:
Nie powinienes zmieniac obiektu imputu. Exterminate powinno przyjmowac liste1, Nie zmieniac jej, zbudowac i zwrocic druga liste

Reply:
To rozwiazanie jest bardzo zagmatwane. Rzuc okiem na to
```
public List<Integer> exterminate(List<Integer> number) {

        List<Integer> result = Lists.newArrayList();

        for (Integer integer : number) {
            if (integer % 2 == 0) {
                result.add(integer);
            }
        }
        return result;
    }
```